pkg/cluster/spec/grafana.go

```diff
@@ -130,14 +132,19 @@ func (i *GrafanaInstance) InitConfig(e executor.Executor, clusterName, clusterVe
 		return err
 	}
 
+	// initial dashboards/*.json
+	if err := i.initDashboards(e, i.InstanceSpec.(GrafanaSpec), paths, clusterName); err != nil {
+		return errors.Annotate(err, "initial dashboards")
+	}
+
 	// transfer dashboard.yml
 	fp = filepath.Join(paths.Cache, fmt.Sprintf("dashboard_%s.yml", i.GetHost()))
 	if err := config.NewDashboardConfig(clusterName, paths.Deploy).ConfigToFile(fp); err != nil {
 		return err
 	}
-	dst = filepath.Join(paths.Deploy, "conf", "dashboard.yml")
-	if err := e.Transfer(fp, dst, false); err != nil {
-		return err
+	dst = filepath.Join(paths.Deploy, "provisioning", "dashboards", "dashboard.yml")
+	if err := i.TransferLocalConfigFile(e, fp, dst); err != nil {
+		return errors.Annotate(err, "transfer dashboard.yml")
```

Comment:
Prefer to annotate the configuration path in `TransferLocalConfigFile` to avoid using `errors.Annotate` everywhere.